app/Jobs/Backup/RestoreDatabaseJob.php

```
@@ -42,10 +56,18 @@ public function handle(): void
         $this->backupFile->status = BackupFileStatus::RESTORE_FAILED;
+        $this->backupFile->message = Str::limit($e->getMessage(), 1000);
         $this->backupFile->save();
         $this->broadcastFileUpdate();
-        ServerLog::log($this->database->server, 'restore-database-failed', $e->getMessage());
+        ServerLog::log($server, 'restore-database-failed', $e->getMessage());
+        Notifier::send($server, new RestoreFailed($server, $this->backupFile));
+
+        try {
+            $server->os()->deleteFile($this->backupFile->tempPath());
+        } catch (Throwable) {
+        }
```

PHPMD (2.15.0)

[error] Avoid using empty try-catch blocks in failed. (undefined)

(EmptyCatchBlock)